frontend/src/Editor/QueryManager/QueryEditors/TooljetDatabase/util.js

```diff
@@ -0,0 +1,36 @@
+import { get } from 'lodash';
+/**
+ * Checks if the queryOptions object contains a filter with an 'eq' (equal) operator and a value equal to '{{null}}'.
+ *
+ * @function hasEqualWithNull
+ * @param {Object} queryOptions - The query options object to check for the presence of the specified filter.
+ * @property {Object} queryOptions.list_rows.where_filters - An object containing the filters to be checked.
+ * @returns {boolean} - Returns true if the specified filter is found, false otherwise.
+ *
+ * @example
+ * const queryOptions = {
+ *   list_rows: {
+ *     where_filters: {
+ *       filter1: {
+ *         operator: 'eq',
+ *         value: '{{null}}',
+ *       },
+ *     },
+ *   },
+ * };
+ *
+ * const result = hasEqualWithNull(queryOptions); // true
+ */
+export const hasEqualWithNull = (queryOptions) => {
+  const filters = get(queryOptions, 'list_rows.where_filters');
```

Comment:
We need to do this check for the other operations where filters are applied, not only for list_rows.